test/gui/shared/steps/steps.py

```diff
@@ -1485,3 +1493,18 @@ def step(context, folderName):
     waitForObject(newAccount.SELECT_REMOTE_DESTINATION_FOLDER_WIZARD)
     newAccount.selectARootSyncDirectory(folderName)
     clickButton(waitForObject(newAccount.ADD_FOLDER_SYNC_CONNECTION_NEXT_BUTTON))
+
+
+@Then("the default sync directory should be created in local file system")
+def step(context):
+    newAccount = AccountConnectionWizard()
+    default_sync_path = waitForObject(newAccount.CHOOSE_LOCAL_DOWNLOAD_DIRECTORY)[
+        'displayText'
+    ]
+    folder_name = str(default_sync_path)
+    clickButton(waitForObject(newAccount.CONF_SYNC_MANUALLY_RADIO_BUTTON))
+    clickButton(waitForObject(newAccount.NEXT_BUTTON))
+    x = squish.waitFor(
+        lambda: os.path.exists(folder_name), context.userData['maxSyncTimeout'] * 500
+    )
+    test.xcompare(True, x)
```

Comment:
should avoid using single char as variable name
```suggestion
    exists = squish.waitFor(
        lambda: os.path.exists(folder_name), context.userData['maxSyncTimeout'] * 500
    )
    test.xcompare(True, exists)
```